js/src/utilities/transform.course.js

```diff
@@ -61,6 +61,7 @@ const transformCourse = (node, data) => {
     })),
     courses_pages: coursesPages.map((coursesPage) => transformCoursesPage({ data: coursesPage })),
     first_lesson_url: firstLessonUrl,
+    progress: fields.getTextValue(node, 'progress'),
```

Comment:
is it a string value or number value?

Reply:
I just saw there is a `getNumberValue` in the utilities that can suit it better although I am handling it as string so I have three concepts
* empty string: progress is not enabled in the course
* "0": no progress
* non-empty and non-zero: some progress
*  
Also the material UI components seems to accept strings fine.

Do you think `progressEnabled` should be a different property?

Reply:
We've used null/empty values elsewhere to transmit whether a feature is enabled or not, so I don't think you need to add another field for progress enabled. As to whether it's a string or number, given the LinearProgress element seems to expect a number I would use that - and use a null value instead of an empty string for the case in which it's not enabled.

Reply:
Fixed. Now when progress is undefined there is no progress bar.